src/CorpusLoaders.jl

```diff
@@ -13,14 +13,13 @@ export load
 
 export WikiCorpus, SemCor, Senseval3, IMDB, Twitter
 
-function __init__()
     include("WikiCorpus_DataDeps.jl")
     include("SemCor_DataDeps.jl")
     include("SemEval2007Task7_DataDeps.jl")
     include("Senseval3_DataDeps.jl")
     include("IMDB_DataDeps.jl")
     include("Twitter_DataDeps.jl")
-end
```

Comment:
The DataDeps registrations need to be run from within  `__init__` to work when precompiled.
Thoough they can be transitions to something onther than `include` in there special files.
E.g. could do something more like what is done in Embeddings.jl
https://github.com/JuliaText/Embeddings.jl/blob/master/src/Embeddings.jl#L17

Or just make one file called `init` that defines the whole `__init__` function,
all in one place

Reply:
I did this on purpose, to check whether this is affecting the error in travis. Now, I saw that it has not shown that error which I stated earlier. I will try to make a separate file for init

Reply:
I think something subtle changed about `include`  when used inside a function in 1.0